datasets/tmu_gfm_dataset/README.md

```diff
@@ -0,0 +1,175 @@
+---
+annotations_creators:
+- crowdsourced
+language_creators:
+- machine-generated
+languages:
+- en
+licenses: 
+- unknown
+multilinguality:
+- monolingual
+size_categories:
+- 1K<n<10K
+source_datasets:
+- original
+task_categories:
+- conditional-text-generation
+task_ids:
+- other
+---
+
+# Dataset Card for TMU-GFM-Dataset
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** [N/A]
+- **Repository:** https://github.com/tmu-nlp/TMU-GFM-Dataset
+- **Paper:** [SOME: Reference-less Sub-Metrics Optimized for Manual Evaluations of Grammatical Error Correction](https://www.aclweb.org/anthology/2020.coling-main.573.pdf)
+- **Leaderboard:** [N/A]
+- **Point of Contact:** Check the paper.
+
+### Dataset Summary
+
+Authors collected manual evaluations for the grammaticality, fluency, and meaning preservation of the system outputs of 1,381 sentences from CoNLL 2013.
+To collect the manual evaluations for various system outputs, each source sentence was corrected by the following five typical systems: statistical machine translation (SMT) (Grundkiewicz and Junczys-Dowmunt, 2018), recurrent neural network (RNN) (Luong et al., 2015), convolutional neural network (CNN) (Chollampatt and Ng, 2018), self-attention network (SAN) (Vaswani et al., 2017), and SAN with copy mechanism (SAN+Copy) (Zhao et al., 2019).
+Manual evaluation for the grammaticality, fluency, and meaning preservation were assigned to a total of 4,223 sentences.
+
+### Supported Tasks and Leaderboards
+
+Grammatical Error Correction
+
+### Languages
+
+English
+
+## Dataset Structure
+
+### Data Instances
+
+An example from the TMU-FGM-Dataset test set looks as follows:
+
+```
+Afterall there will be an endless battle between the technology and human mentality.,"After all, there will be an endless battle between the technology and human mentality.","3,4,3,4,3","3,4,3,4,3","3,4,4,4,3","lstm,cnn",3.4,3.4,3.6
+```
```

Comment:
here we expect a dictionary that you can get from using `load_dataset` and taking the first sample `dataset["train"][0]` for example.

Can you change that please ?

Reply:
Absolutely! Sorry for my misunderstanding. I'll change that in 24 hours.

Reply:
I think I could make the necessary corrections. Is this correction appropriate?

The order of keys has changed from the dataset description, probably because it is an unordered dictionary. Is it OK, or should I use OrderedDict or some other solutions?

Reply:
It's fine this way thanks :)